README.md

```diff
@@ -108,11 +108,35 @@ for users to navigate Julia's many varied packages.
 As long as the package is not yet registered, renaming the package from
 `OldName.jl` to `NewName.jl` is reasonably straightforward:
 
-* Rename the GitHub repository to `NewName.jl`
+* [Rename the GitHub repository](github-rename) to `NewName.jl`
 * Rename the file `src/OldName.jl` to `src/NewName.jl`
 * Rename the top-level module to `NewName`
 * Update tests, documentation, etc, to reference the new name
 
+#### How do I rename an existing registered package?
+Technically, you can't rename a package once registered, as this would break existing users.
+But you can re-register the package again under a new name with a new UUID.
+Which has basically the same effect.
+
+ - Follow the instructions above for renaminging a package: rename on GitHub, rename files etc.
+ - Generate a new UUID for the Project.toml
+ - Increment the version in the Project.toml as a breaking change.
+ - [Register](#registering-a-package-in-general) it as if it were a new package
+ - Comment on the PR, that this is a rename.
+ - It will have to go though the normal criteria for registring a new package.
+    - In particular, even if you get it merged manually, it will need to wait 3 days from the PR being openned.
+    - This gives others and youself the chance to point out any naming issues.
```

Comment:
Should be “yourself” instead of “youself”

Reply:
```suggestion
    - This gives others and yourself the chance to point out any naming issues.
```